lib/urbanairship/devices/subscription_lists.rb

```diff
@@ -0,0 +1,46 @@
+require 'urbanairship'
+
+
+module Urbanairship
+  module Devices
+    class SubscriptionLists
+      include Urbanairship::Common
+      include Urbanairship::Loggable
+      SUBSCRIBE = "subscribe"
+
+      def initialize(client: required('client'))
+        @client = client
+      end
+
+      def subscribe(list_id, email_addresses)
+        subscribe_payload = payload(SUBSCRIBE, list_id, email_addresses)
```

Comment:
Do you need to raise an error if `list_id` or `email_addresses` aren't supplied:

```ruby
        fail TypeError, 'list_id musy be provided' unless list_id.is_a? String
```

Reply:
Thank you, I will take care of that!